### ISSUE TITLE
Support for Second Life

### ISSUE THREAD

**Issue by kellabyte** [polygraphene/ALVR#184]
Second Life would be a much more amazing VR world than some of the Oculus Go specific ones or Steam VR ones. Would you support this game at all? There's really no game that compares in it's category. I know it's not a Steam VR game but I would really love to be able to use the Oculus Go with Second Life. There's just so much virtual world in Second Life, it's amazing.

**Comment by polygraphene** [comment]
Are you talking about Sansar?
It is not published in Steam, but it may depend on SteamVR. If so, we can use ALVR without any modifications.
I will check it when I have a time.

**Comment by kellabyte** [comment]
No, I'm talking about Second Life. Sansar has almost no people on it. Second Life is a giant world with half a million people signed in at any time. I recently got a VR headset and I would love to continue playing Second Life with VR!

All these other VR "worlds" are nowhere near the same as Second Life. Second Life is like the web of VR almost, except it just doesn't support VR headsets! It's a huge world though!

**Comment by polygraphene** [comment]
Hmm. I got it.
If you want to play non-VR game with VR, you should check vorpX.
https://www.vorpx.com/

Fortunately, vorpX can work with ALVR. #51
